grakn-graql/src/main/java/ai/grakn/graql/internal/query/QueryAnswer.java

```diff
@@ -60,7 +60,7 @@
  */
 public class QueryAnswer implements Answer {
 
-    private final ImmutableMap<Var, Concept> map;
+    private final Map<Var, Concept> map;
```

Comment:
I think the point of having an explicit `ImmutableMap` here was that you open the class and you can expect it to be immutable without looking at the details.

Reply:
I'll change in the next PR